pkgs/net-lib/net/sendurl.rkt

```diff
@@ -93,24 +78,23 @@
 ;; send-url : str [bool] -> void
 (define (send-url url-str [separate-window? separate-by-default?]
                   #:escape? [escape? #t])
-  (define stype (force systype))
   (unless (string? url-str)
     (error 'send-url "expected a string, got ~e" url-str))
   (let ([url-str (if escape? (escape-url url-str) url-str)])
     (if (procedure? (external-browser))
       ((external-browser) url-str)
-      (case stype
-        [(macosx)  (send-url/mac url-str)]
-        [(windows) (send-url/win url-str)]
-        [(unix)    (send-url/unix url-str (force separate-window?))]
+      (case (system-type)
+        ['macosx  (send-url/mac url-str)]
```

Comment:
This happens to work because of how the reader abbreviation for quote works, but these should be parenthesized, not quoted.

Reply:
Thanks, updated.